src/EditorFeatures/Text/TextEditorFeatures.csproj

```diff
@@ -58,6 +58,8 @@
     <InternalsVisibleToTest Include="Roslyn.Services.Editor.VisualBasic.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.VisualStudio.Services.UnitTests" />
     <InternalsVisibleToTest Include="RoslynETAHost" />
+    <InternalsVisibleToFSharp Include="FSharp.Editor" />
```

Comment:
You're also in the test section.  Ideally all your stuff would always be next to the TypeScript stuff.